components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1594,35 +1599,37 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
                                         )}
                                     </div>
                                 )}
-                                <OverlayTrigger
-                                    placement='top'
-                                    delayShow={Constants.OVERLAY_TIME_DELAY}
-                                    trigger={Constants.OVERLAY_DEFAULT_TRIGGER}
-                                    overlay={(
-                                        <Tooltip id='post-priority-picker-tooltip'>
-                                            <FormattedMessage
-                                                id={'post_priority.remove'}
-                                                defaultMessage={'Remove {priority}'}
-                                                values={{priority: this.props.draft.metadata!.priority!.priority}}
-                                            />
-                                        </Tooltip>
-                                    )}
-                                >
-                                    <button
-                                        type='button'
-                                        className='close'
-                                        onClick={this.handleRemovePriority}
+                                {!this.props.shouldShowPreview && (
```

Comment:
adding this into the props directly makes it a bit hard to read. Would you mind storing this in a var beforehand?